src/contracts/CrossChainForwarder.sol

```diff
@@ -56,14 +63,23 @@ contract CrossChainForwarder is OwnableWithGuardian, ICrossChainForwarder {
    */
   constructor(
     ForwarderBridgeAdapterConfigInput[] memory bridgeAdaptersToEnable,
-    address[] memory sendersToApprove
+    address[] memory sendersToApprove,
+    RequiredConfirmationsByReceiverChain[] memory requiredConfirmationsByReceiverChain
   ) {
     _configureForwarderBasics(
       bridgeAdaptersToEnable,
       new BridgeAdapterToDisable[](0),
       sendersToApprove,
       new address[](0)
     );
+    _updateRequiredConfirmationsForReceiverChain(requiredConfirmationsByReceiverChain);
```

Comment:
Why this is not part of `_configureForwarderBasics()`? It is clearly a basic configuration